stress/v2/ponyExpress/commune.go

```diff
@@ -0,0 +1,65 @@
+package ponyExpress
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/influxdata/influxdb/models"
+)
+
+// Communes are a method for passing points between InsertStatements and QueryStatements.
+
+type commune struct {
+	ch          <-chan string
+	storedPoint models.Point
+}
+
+// NewCommune creates a new commune with a buffered chan of length n
+func newCommune(n int) (*commune, chan<- string) {
+	ch := make(chan string, n)
+	c := &commune{
+		ch: ch,
```

Comment:
This function is unexported, so all callers must have access to the unexported fields on `commune`. Given that, I think you could simplify this to:

```go
func newCommune(n int) *commue {
  return &commune{ch: make(chan string, n)}
}
```